daemon/daemon.go

```diff
@@ -508,7 +515,32 @@ func (d *daemon) createEvent(ev event.Event) error {
 	return nil
 }
 
+
+
+
+// todo: DOES NOT CREATE EVENT, IT CREATES CONFIGURATION FILE TO CREATE EVENT  !!!!!!!!!!!!
+
 func (d *daemon) CreateEvent(req *pb.CreateEventRequest, resp pb.Daemon_CreateEventServer) error {
+	// Since there are more than one goroutines
+	// there should be synchronization of them
+	// to be able to send data back to client
+	// goroutines should be waited
+	var wg sync.WaitGroup
+	wg.Add(2)
+	var m sync.Mutex
+	m.Lock()
```

Comment:
What is the purpose of the lock when it is unlocked immediately below (line 535)?

Reply:
Yeah, it is left from other tests. removed.